codemeta.json

```diff
@@ -34,9 +34,9 @@
     "developmentStatus": [
         "https://www.repostatus.org/#active",
         {
-            "@id":  "https://w3id.org/research-technology-readiness-level#Level9Proven",
+            "@id":  "https://w3id.org/research-technology-readiness-levels#Level9Proven",
             "http://www.w3.org/2004/02/skos/core#inScheme": {
-                "@id":  "https://w3id.org/research-technology-readiness-level"
+                "@id":  "https://w3id.org/research-technology-readiness-levels"
```

Comment:
Do we still need the `http://www.w3.org/2004/02/skos/core#inScheme` bit?

Reply:
No, that'd be too verbose/redundant to include explicitly. This context is present in the knowledge graph when codemetapy runs the validation.

It's only when you run the shacl yourself that it's missing and fails on this. I'll still need to add a workaround for that.